ObjectCache_Environment.php

```diff
@@ -112,11 +112,11 @@ private function create_addin() {
 					// we should try to apply ours
 					// (in case of missing permissions deletion could fail)
 				} else if ( !$this->is_objectcache_old_add_in() ) {
-					if ( isset( $_GET['page'] ) )
-						$url = 'admin.php?page=' . $_GET['page'] . '&amp;';
+					if ( isset( Util_Request::get_string( 'page' ) ) )
```

Comment:
Use `! empty()`.  You may want to set a variable to avoid calling twice.

Reply:
Changed to ! empty and created a variable to avoid 2 function calls.